### ISSUE TITLE
NbTabsetComponent not exported?

### ISSUE THREAD

**Issue by Dikymon** [akveo/nebular#1353]
### Issue type

**I'm submitting a ...**  (check one with "x")

* [ x ] bug report
* [ ] feature request

### Issue description

**Current behavior:**
I need to add a custom feature to NbTabsetComponent, but it's not exported? Though i don't see why not, when following the export trail. Am I not seeing something obvious?

**Expected behavior:**
That I can succesfully extend NbTabsetComponent without getting 
`Module '".../node_modules/@nebular/theme/index"' has no exported member 'NbTabsetComponent'.`

**Steps to reproduce:**

export class TabsComponent extends NbTabsetComponent implements OnInit {
...
}

theme/index.ts
```
export * from './components/tabset/tabset.module';

```

theme/components/tabset/tabset.module.ts
```
const NB_TABSET_COMPONENTS = [
  NbTabsetComponent,
  NbTabComponent,
];

...

exports: [
    ...NB_TABSET_COMPONENTS,
]

```

**Comment by Dikymon** [comment]
An addendum to my own issue: Not all components are exported - by design. This is unfortunate, as we need to access these for both custom components and integration testing.

I ended up adding an npm postinstall script that would insert the needed exports in @nebular/theme/index.d.ts

**Comment by nnixaa** [comment]
fixed in 4.0
